modules/service/src/main/resources/job_templates/datax/readers/hive.tpl

```diff
@@ -1,23 +1,24 @@
-{
-    "name": "hdfsreader",
-    "parameter":{
-        "datasource": #{datasourceId},
-        "authType": #{authType},
-        "path": #{hiveDataPath},
-        "column": ${column|["*"]},
-        "fileType": #{fileType|TEXT},
-        "encoding": #{encoding|UTF-8},
-        "fieldDelimiter": #{fieldDelimiter|\u0001},
-        "partitionValues": #{partitions},
-        "hiveTable": #{table},
-        "hiveDatabase": #{database},
-        "defaultFS": #{defaultFS|<defaultFS>},
-        "hadoopConfig": ${hadoopConfig|"<hadoopConfig>"},
-        "haveKerberos":#{haveKerberos|false},
-        "kerberosPrincipal": #{kerberosPrincipal},
-        "kerberosKeytabFilePath": #{kerberosKeytabFilePath},
-        "ldapUserName": #{ldapUserName},
-        "ldapUserPassword": #{ldapUserPassword},
-        "nullFormat": #{nullFormat|\\N}
-    }
-}
\ No newline at end of file
+{
+    "name": "hdfsreader",
+    "parameter":{
+        "datasource": #{datasourceId},
+        "authType": #{authType},
+        "path": #{hiveDataPath},
+        "column": ${column|["*"]},
+        "fileType": #{fileType|TEXT},
+        "encoding": #{encoding|UTF-8},
+        "hivePrincipal": "hadoop/_HOST@EXAMPLE.COM",

```

Comment:
Use "hadoop/_HOST@EXAMPLE.COM" as default value for “hivePrincipal"

Reply:
The  task  reads the hive with the wrong hivePrincipal if user define the different principle from the value   "hadoop/_HOST@EXAMPLE.COM".